### PR TITLE
Updated imports to use standard Python packaging.

### PR DESCRIPTION
**Pull request by tameware**

### CODE REVIEW

**Comment by tameware** [comment]
I've confirmed that this works when deployed to GCP. It's running in our dev environment now.

File: README.gcp.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ make run_local_tests
 
 ```
 pip3 install -r requirements.txt
-cd ~/bridge-hackathon/DDS/src
-python3 api.py
+cd ~/bridge-hackathon/DDS/
```
Comment (suokko) [inline README.gcp.md:59]: The last slash can be dropped

Comment (tameware) [reply README.gcp.md:59]: It certainly can be! I've noticed that often documentation includes it, presumably to emphasize that it's a directory and not a file. Not necessary for clarification when the command is *cd*, but consistency is a virtue.

**Comment by tameware** [comment]
> It would be nice to add link to the answer in commit message.

Which answer, the one on StackOverflow? Alex didn't want to post there, so I'll end up answering my own question.

I've updated the commit message per what I think you're suggesting.

**Comment by suokko** [comment]
> > It would be nice to add link to the answer in commit message.
> 
> Which answer, the one on StackOverflow? Alex didn't want to post there, so I'll end up answering my own question.
> 
> I've updated the commit message per what I think you're suggesting.

Has the fellow of python Alex joined Hackathon?
That would make me more comfortable to use python if there is someone with a lot python knowledge.

Sorry. I suspect it was an stackoverflow answer. In that case it would be nice to has first line of commit message a short in the point header about change. Followed by an empty line. Then a more detailed explanation how this changes makes sure module internal imports and external imports. Also credit for the idea should go to the detailed part of commit message.

File: README.gcp.md (hunk 1 above)
Comment (tameware) [reply README.gcp.md:59]: I've read that `import *` is frowned upon in production code.

Comment (tameware) [reply README.gcp.md:59]: Commit message updated.

**Comment by tameware** [comment]
Alex Martelli has signed up fr the Hackathon. Whether he has time to join actively is another matter! He's the author of a few of  O'Reilly's Python Books and also a few Bridge World articles.

**Pull request merged**